src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ForAll.txt

```diff
@@ -109,3 +109,26 @@ Error({Kind:ErrorKind.Numeric})
 
 >> ForAll(ParseJSON("[""1"",""a""]"), If(Text(ThisRecord) = "1", Value(ThisRecord) * 2, Value(ThisRecord)))
 Error({Kind:ErrorKind.InvalidArgument})
+
+>> ForAll(Table(Blank()),ThisRecord)
+Table(Blank())
+
+>> ForAll(Table(Blank()),Coalesce(ThisRecord))
+Table(Blank())
+
+>> ForAll([1,2,3], If(Value <> 2, {a:Value}))
+Table({a:1},Blank(),{a:3})
+
+>> Concat(ForAll([1,2,3,4,5],If(Value = 1 Or Value = 3 Or Value = 5, {a:Value},Value = 4, {a:Blank()},Blank())),If(IsBlank(ThisRecord), "<<blank record>>",IsBlank(ThisRecord.a), "<<blank value>>",Text(ThisRecord.a)),",")
+"1,<<blank record>>,3,<<blank value>>,5"
+
+>> Concat(ForAll([1,2,3],If(Value=1 Or Value=3,Sequence(Value),Blank())),If(IsBlank(ThisRecord),"<<blank record>>",IsBlank(Value),"<<blank value>>",$"Table with{CountRows(Value)}elements"),", ")
+"Table with1elements, <<blank value>>, Table with3elements"
```

Comment:
This is covering the same codepath as the test on line 133; your call whether to keep it or remove this one